site/content/blogs/apprunner-waf.en.md

```diff
@@ -0,0 +1,83 @@
+# Using WAF With App Runner in Copilot
+
+Posted On: Feb 08, 2023
+
+**Siddharth Vohra**, Software Development Engineer, AWS App Runner
+
+You can now associate your AWS WAF WebACLs with your App Runner service - all in Copilot!
+
+We posted these step in our [GitHub "Show and tell" discussion section](LINK TO BE ADDED) as well! If you have any questions, feedbacks or requests that are related to AppRunner's WAF support, feel free to drop a comment there!
+
+AWS Web Application Firewall (WAF) helps you monitor requests that are forwarded to your web applications and allows you to control access to your content.To do this, you need to create a WAF Web Application Control List (ACL) which can contain many rule 
+options for your application. See [here](https://docs.aws.amazon.com/waf/latest/developerguide/web-acl-creating.html) for more on creating your own Web ACL. Once you’ve created a Web ACL through AWS WAF Console or the AWS CLI, copy and save its ARN. To use the WAF ACL with your App Runner service managed by Copilot, follow the steps below.  
+
+Step 1: If you don’t have an App Runner service created/deployed, run `copilot svc init --type "Request-Driven Web Service"` to create and configure an App Runner service.
+
+Step 2: Go to the service's directory (users/\<your username\>/copilot/\<your App Runner service name\>). If you don't have a folder called 'addons' in this directory, create a new 'addons' folder. In the addons folder, create two new files: `ar_waf-addon.yml` and `addons.parameters.yml`. Your folders would now look like this:  
+
+  ```term
```

Comment:
Can we do something like [this](https://github.com/aws/copilot-cli/blob/212611525869a81396c6f1b57a479cc1ea5317be/site/content/docs/manifest/lb-web-service.en.md?plain=1#L3) so that these code snippets can be combined with different titles? Same for the `waf.yaml` file below.

Reply:
this is done ✔️